### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by mc-usds**
## Background

Updating to pull request template to ask for information that will make it easier to review code changes from external teams.

#### Unique to this PR
N/A

#### Applies to all PRs
N/A
- [ ] Appropriate test coverage & logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)

### CODE REVIEW

**Comment by mc-usds** [comment]
Trying out the code review group. @department-of-veterans-affairs/code-reviewers-for-external-teams-group

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
-## Background
-<!-- What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary -->
+## Description of change
+Please include a description of the change. What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary.
```
Comment (lihanli) [inline .github/PULL_REQUEST_TEMPLATE.md]: can we leave the html comments in? that way the user doesn't need to delete it every time

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
 
-## Definition of Done
+## Testing done
+Please describe testing done to verify the changes.
```
Comment (lihanli) [inline .github/PULL_REQUEST_TEMPLATE.md]: for some small PRs testing done and testing planned can just be accomplished through automated tests, is it necessary to have this added to every template?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jkassemi) [inline .github/PULL_REQUEST_TEMPLATE.md]: Do we want to call out links to relevant issues?

Comment (jkassemi) [reply .github/PULL_REQUEST_TEMPLATE.md]: It's hard to come up with a universal set of requirements for this, and I fully anticipate making changes as we go along; some things to stew on:

1) does this change modify integrations with third party systems?
2) is there a feature flag for this change, and if so, does the devops team need to e made aware of varying configuration for each environment?
3) are there any backwards-incompatible changes for consumers of the API, and have all API modifications been reflected in the swagger documentation?
4) are all requests asynchronous or the reasons for them not being so documented?
5) are there dashboards and associated instrumentation in place for this change? Have alerts been configured? 
6) Which alerts would indicate an issue with this functionality?

Comment (mc-usds) [reply .github/PULL_REQUEST_TEMPLATE.md]: Will change to be html comments.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (jkassemi) [reply .github/PULL_REQUEST_TEMPLATE.md]: @lihanli can you point to an example of a PR where at least local acceptance testing is unnecessary?

Comment (mc-usds) [reply .github/PULL_REQUEST_TEMPLATE.md]: I think it's good to have as a to make sure that testing is thought about and executed. In the case where the change was be tested via existing automated tests or new automated tests, the submitter can say that.

Comment (lihanli) [reply .github/PULL_REQUEST_TEMPLATE.md]: @jkassemi this is an example, https://github.com/department-of-veterans-affairs/vets-api/pull/2202, this section can be useful for more complex PRs though and we can write automated tests for small changes

Comment (jkassemi) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thanks @lihanli! Sounds like @mc-usds's approach will work well in cases like that, for sure.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
+## Testing planned
+<!-- Please describe testing planned. -->
+
+## Acceptance Criteria (Definition of Done)
 
+#### Unique to this PR
+<!-- This would be a good place to include feature flag check item and info, specific dashboards and instrumentation check item and info -->
 - [ ] _Replace this line with individual tasks unique to your PR_
 
 #### Applies to all PRs
```
Comment (jholton) [inline .github/PULL_REQUEST_TEMPLATE.md]: With testing being covered above, should we remove the line below or change it to "Appropriate logging"?

Comment (mc-usds) [reply .github/PULL_REQUEST_TEMPLATE.md]: Updated the checklist item to be "Appropriate logging"

**Pull request merged**